Readme.md

```diff
@@ -48,7 +48,8 @@ This is done by setting the `id` of each event to `-` and returning `204 : No in
 
 Install npm dependencies using `yarn` and run `yarn test` to run the tests. 
 To build it locally you need to use `NODE_ENV='test' TASKCLUSTER_ROOT_URL='localhost:12345' node src/main.js server` 
-This will use the pulse credentials in `user-config.yml`. Set `DEBUG: events:*` for additional debugging information.
+This will use the pulse credentials in the test profile of `user-config.yml` (see user-config-example.yml). Set `PULSE_VHOST: /`  to use internally.
```

Comment:
In these cases, `DEBUG` is an env var so it should be set with `=` and `PULSE_VHOST` should be set in user-config so it is `pulse.vhost: '/'`. What do we mean by internally btw?

Reply:
Yeah it should be `=` .Sorry i didnt catch that earlier.
I mean when using with `pulse.mozilla.org`